Database/ingestion.py

```diff
@@ -133,6 +133,7 @@ def check_user_exists(user_id: int) -> bool:
                     user_details = extract_user_details(msg)
                     print(user_details)
                     user_id = user_details["user_id"]
+                    user_age=age_from_dob(user_details['dob_date'])
```

Comment:
could you add spaces after equal signs unless they are parameters inside methods and functions